miniapp/USERGUIDE.md

```diff
@@ -27,9 +27,23 @@ dependency {
 }
 ```
 
-### #2 Set your App Id, Subscription Key, & Base URL
+### #2 Configure SDK settings in AndroidManifest.xml
 
-We don't currently host a public API, so you will need to provide your own Base URL for API requests.
+The SDK is configured via manifest meta-data, the configurable values are:
+
+| Field                        | Datatype| Manifest Key                                         | Optional   | Default   |
+|------------------------------|---------|------------------------------------------------------|------------|---------- |
+| Base URL                     | String  | `com.rakuten.tech.mobile.miniapp.BaseUrl`            | ❌         | 🚫        |
+| Testbox Config               | boolean | `com.rakuten.tech.mobile.miniapp.IsTestMode`         | ✅         | `false`   |
+| Host App Version             | String  | `com.rakuten.tech.mobile.miniapp.HostAppVersion`     | ❌         | 🚫        |
+| Host App Info                | String  | `com.rakuten.tech.mobile.miniapp.HostAppInfo`        | ✅         | 🚫        |
```

Comment:
This needs to be updated to be reflective of what this information is specifically being used for. Hence,  
`Host App Info` ->  `Host App User Agent Info`